syncer/ddl.go

```diff
@@ -40,7 +41,7 @@ func (s *Syncer) parseDDLSQL(sql string, p *parser.Parser, schema string) (resul
 	sql = utils.TrimCtrlChars(sql)
 
 	// check skip before parse (used to skip some un-supported DDLs)
-	ignore, err := s.skipQuery(nil, nil, sql)
+	ignore, _, err := s.skipQuery(nil, nil, sql)
```

Comment:
if here a binlog filter using RE to filter a SQL, `splitAndFilterDDL` will not be called and we won't track that DDL

Reply:
What does RE means?